tap_hubspot/hubspot.py

```diff
@@ -238,55 +248,44 @@ def get_contacts(
             MANDATORY_PROPERTIES[f"{obj_type}"],
         )
 
-        while True:
-            deals = {}
-            i = 0
-
-            for rec, ts in gen:
-                rec_id = rec["id"]
-                # ensure that there is a default entry for all contacts/companies
-                rec["associations"] = {
-                    # fit the legacy format to be backwards compatible:
-                    # "results": [{ "id": "<id>" }]
-                    "deals": {"results": []},
-                    "companies": {"results": []},
+        for chunk in chunker(gen, 10):
```

Comment:
change to 100 for this batch should be fine. I use 50 in my PR because the history endpoint only accepts 50 as the size